### ISSUE TITLE
InboundSendEmailHandler do not send mail when no account needed

### ISSUE THREAD

**Issue by maclee99** [Mikroservices/Smtp#20]
my workaround:
`
case .okAfterHello:

            if self.shouldInitializeTls() {
                self.send(context: context, command: .startTls)
                self.currentlyWaitingFor = .okAfterStartTls
            } else {
                // self.send(context: context, command: .beginAuthentication)
                // self.currentlyWaitingFor = .okAfterAuthBegin
                if !self.serverConfiguration.username.isEmpty {
                    self.send(context: context, command: .beginAuthentication)
                    self.currentlyWaitingFor = .okAfterAuthBegin
                } else {
                    self.send(context: context, command: .mailFrom(self.email.from.address))
                    self.currentlyWaitingFor = .okAfterMailFrom
                }

            }
`

**Comment by mczachurski** [comment]
Implemented in version 3.0.0.

**Comment by mczachurski** [comment]
Example code: 

```swift
let app = Application(env)
defer { app.shutdown() }

app.smtp.configuration.host = "smtp.server"
app.smtp.configuration.signInMethod = .anonymous
app.smtp.configuration.secure = .ssl

try configure(app)
try app.run()
```
